hadoop-ozone/dist/src/main/smoketest/ozone-lib/shell.robot

```diff
@@ -61,3 +61,21 @@ Create Key
     ${output} =    Execute          ozone sh key put ${key} ${file}
                    Should not contain  ${output}       Failed
     Log            Uploaded ${file} to ${key}
+
+Verify Bucket EC Replication Config
+    [arguments]    ${volume}    ${encoding}    ${data}    ${parity}    ${chunksize}
+    ${result} =    Execute                      ozone sh bucket list ${volume} | jq -r '.[] | select(.name | contains("${prefix}ec")) | .replicationConfig.replicationType, .replicationConfig.codec, .replicationConfig.data, .replicationConfig.parity, .replicationConfig.ecChunkSize'
```

Comment:
I think directly passing bucket is more reusable, instead of relying on suite variable and naming convention.

```suggestion
    [arguments]    ${bucket}    ${encoding}    ${data}    ${parity}    ${chunksize}
    ${result} =    Execute                      ozone sh bucket info ${bucket} | jq -r '.replicationConfig.replicationType, .replicationConfig.codec, .replicationConfig.data, .replicationConfig.parity, .replicationConfig.ecChunkSize'
```

Reply:
I did not mean to carry $prefix into this part of the code. I missed that then copying and pasting I think. Your solution is better so I have committed them and will let the CI run and hopefully it passes again.